shared/django_apps/db_settings.py

```diff
@@ -112,7 +112,7 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.postgresql",
+        "ENGINE": "psqlextra.backend",
```

Comment:
Updating these and not just dummy_settings.py so they are picked up by api + worker